tools/Makefile

```
@@ -10,7 +10,9 @@ tools := \
 	pokemon_animation \
 	pokemon_animation_graphics \
 	scan_includes \
-	stadium
+	stadium \
+	gfx \
```

`gfx` is listed in alphabetical order. (You should move `make_patch` in order too.)
```suggestion
	gfx \
```

---

https://github.com/pret/pokecrystal/pull/882/commits/a434da8d40e5f3403d94dd9053ca0a42e1d85a63 should resolve this.